packages/components/src/ConfirmDialog/ConfirmDialog.component.js

```diff
@@ -39,24 +41,25 @@ function ConfirmDialog({
 	validateAction,
 	cancelAction,
 	progressValue,
+	bodyOverflow = true,
 }) {
 	return (
 		<Modal
 			bsSize={size}
 			show={show}
 			keyboard
-			className={classNames(theme['tc-confirm-dialog'])}
+			className={classNames(theme['tc-confirm-dialog'], bodyOverflow && theme['modal-body-overflow'])}
```

Comment:
those are not ternary expression, this is a simple js expression returning je last element if bodyOverflow is truthy, it may even be faster than creating a new object literal, checking if first key is true and then extracting said key.